skycoin/skywire
# Rebuild frontend

Did you run `make format && make check`? Yes

 Changes:	
- Rebuild frontend after recent frontend changes

How to test this PR:
- Run any integration env
- Check hypervisor UI
